README.md

```diff
@@ -26,62 +26,83 @@ The purpose of this repository is to contain libraries which users can import an
+#### Assets
+
+- [Token](./libs/token/) provides helper functions for the [SRC-20](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_20), [SRC-3](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_3), and [SRC-7](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_7) standards.
+
+#### Access Control and Security
+
 - [Ownership](./libs/ownership/) is used to apply restrictions on functions such that only a single user may call them.
+- [Pausable](./libs/pausable/) allows contracts to implement an emergency stop mechanism.
 - [Reentrancy](./libs/reentrancy) is used to detect and prevent reentrancy attacks.
-- [Signed Integers](./libs/signed_integers/) is an interface to implement signed integers.
+
+#### Cryptography
+
+- [Merkle Proof](./libs/merkle_proof/) is used to verify Binary Merkle Trees computed off-chain.
+
+#### Math
+
 - [Fixed Point Number](./libs/fixed_point/) is an interface to implement fixed-point numbers.
+- [Signed Integers](./libs/signed_integers/) is an interface to implement signed integers.
+
+#### Data Structures
+
 - [Queue](./libs/queue/) is a linear data structure that provides First-In-First-Out (FIFO) operations. 
-- [Token](./libs/token/) provides helper functions for the [SRC-20](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_20), [SRC-3](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_3), and [SRC-7](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_7) standards.
-- [Pausable](./libs/pausable/) allows contracts to implement an emergency stop mechanism.
 
 ## Using a library
 
-To import the Merkle Proof library the following should be added to the project's `Forc.toml` file under `[dependencies]` with the most recent release:
+To import a library, a dependency should be added to the project's `Forc.toml` file under `[dependencies]`. The following shows adding the Merkle Proof Library.
 
 ```rust
 merkle_proof = { git = "https://github.com/FuelLabs/sway-libs", tag = "v0.1.0" }
 ```
 
+> **NOTE:** 
+> Be sure to set the tag to the latest release.
+
 You may then import your desired library in your Sway Smart Contract as so:
 
-```rust
+```sway
 use merkle_proof::<library_function>;
 ```
 
-For example, to import the Merkle Proof library use the following statement:
+For example, to import the `verify_proof()` function use the following statement:
 
-```rust
-merkle_proof::binary_merkle_proof::verify_proof;
+```sway
+use merkle_proof::binary_merkle_proof::verify_proof;
 ```
 
 ## Running Tests
 
-In order to run the tests make sure you are in the tests folder of this repository `sway-libs/tests/<you are here>`.
+There are two sets of tests that should be run: inline tests and sdk-harness tests.
 
-Build the test projects:
+In order to run the inline tests, make sure you are in the `libs/` folder of this repository `sway-libs/libs/<you are here>`.
+
+Run the tests:
 
 ```rust
-forc build
+forc test
```

Comment:
Could you amend the `rust` language declaration above to be `bash` and add one to the code block below too?